src/registry.c

```diff
@@ -762,7 +777,9 @@ parse_model(struct rxkb_context *ctx, xmlNode *model,
 {
     char *name, *description, *brief, *vendor;
 
-    if (parse_config_item(ctx, model, &name, &description, &brief, &vendor)) {
+    if (parse_config_item(ctx, model, &popularity, &name,
+                          &description, &brief, &vendor))
+    {
```

Comment:
```suggestion
                          &description, &brief, &vendor)) {
```

same for the others below

Reply:
Done